### PR TITLE
feature: added making text longer

### PR DESCRIPTION
**Pull request by nurmukhametdaniyar**
### Feature Preview
image

fixes https://github.com/AppFlowy-IO/AppFlowy/issues/2037

---

#### PR Checklist

- [x] My code adheres to the [AppFlowy Style Guide](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/submitting-code/style-guides)
- [x] I've listed at least one issue that this PR fixes in the description above.
- [x] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [x] All existing tests are passing.

### CODE REVIEW

**Comment by LucasXu0** [comment]
Hey, @nurmukhametdaniyar. Can you help to correct the previous l10n too? 

this l10n for this two options should be Improve writing and Make longer.

File: frontend/appflowy_flutter/assets/translations/en.json [hunk 1]
```diff
@@ -364,6 +364,7 @@
       "warning": "⚠️ AI responses can be inaccurate or misleading.",
       "smartEditSummarize": "Summarize",
       "smartEditImproveWriting": "Improve Writing",
```
Comment (LucasXu0) [inline frontend/appflowy_flutter/assets/translations/en.json:366]: Improve writing

File: frontend/appflowy_flutter/assets/translations/en.json [hunk 2]
```diff
@@ -364,6 +364,7 @@
...
+      "smartEditMakeLonger": "Make it Longer",
```
Comment (LucasXu0) [inline frontend/appflowy_flutter/assets/translations/en.json:367]: Make longer

File: frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/openai/widgets/smart_edit_action.dart [hunk 3]
```diff
@@ -27,6 +30,8 @@ enum SmartEditAction {
         return 'Correct this to standard English:\n\n$input';
       case SmartEditAction.improveWriting:
         return 'Rewrite this:\n\n$input';
+      case SmartEditAction.makeItLonger:
+        return 'Make this text longer:\n\n$input';
```
Comment (LucasXu0) [inline frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/openai/widgets/smart_edit_action.dart:34]: By the way, have you tested this prompt?

Comment (nurmukhametdaniyar) [reply frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/openai/widgets/smart_edit_action.dart:34]: Tested it, looks good to me

**Pull request merged**